### ISSUE TITLE
Access control based on auth level

### ISSUE THREAD

**Issue by uahic** [RobotWebTools/roslibjs#439]
Hi there,

is it possible with the current Authentication implementation to gain access to certain services only if the user level is the right one?
I tried implementing my own auth node with service /authentication using the srv type of rosauth but it does not get called at all :/  (Unknown operation: auth),  authenticate is enabled in the launch script

The fields in the msg contain src/dest IP adresses but what I really want to check is against service names.
Basically what I need would be similar to JWT

Thank you

**Comment by uahic** [comment]
I will move this issue to the rosbridge_server issue-section
